u_common.go

```diff
@@ -155,6 +160,20 @@ var (
 	HelloIOS_Auto = HelloIOS_12_1
 	HelloIOS_11_1 = ClientHelloID{helloIOS, "111", nil} // legacy "111" means 11.1
 	HelloIOS_12_1 = ClientHelloID{helloIOS, "12.1", nil}
+
+	HelloEdge_Auto = HelloEdge_85
+	HelloEdge_85   = ClientHelloID{helloEdge, "85", nil}
+
+	HelloExplorer_11 = ClientHelloID{helloExplorer, "11", nil}
+
+	HelloSafari_Auto = HelloSafari_13_1
+	HelloSafari_13_1 = ClientHelloID{helloSafari, "13.1", nil}
+
+	Hello360_Auto = Hello360_7_5
+	Hello360_7_5  = ClientHelloID{hello360, "7.5", nil}
+
+	HelloQQ_Auto = HelloQQ_10_6
+	HelloQQ_10_6 = ClientHelloID{helloQQ, "10.6", nil}
```

Comment:
The other browsers (Chrome and Firefox) are only specified by major version.  However, I have no idea what the release cycle is like for the Chinese browsers (360 and QQ), so I felt it may make more sense to specify both the major and minor version for these.

Reply:
Makes sense to me :+1: